public/src/components/channelManagement/stickyTopBar/stickyTopBar.tsx

```diff
@@ -209,6 +213,9 @@ const StickyTopBar: React.FC<StickyTopBarProps> = ({
               </Button>
             </>
           )}
+          <Button variant="outlined" size="medium" onClick={() => onTestAudit(name, channel)}>
+            <Typography className={classes.buttonText}>Audit details</Typography>
```

Comment:
Tiny thing - but it might be nice to have an icon in the button like we have with the others.
Perhaps this one? - 
`import HistoryIcon from '@mui/icons-material/History';`
And perhaps shorten the copy to just "Audit"

Reply:
yes, added the HistoryIcon and shortened the copy